### PR TITLE
Use act in Justfile to run CI checks locally

### PR DESCRIPTION
**Pull request by peterfication**

### CODE REVIEW

**Comment by peterfication** [comment]
@johmsalas this adds support for act commands. Running the nightly job is also possible.

File: Justfile [hunk 1]
```diff
@@ -2,8 +2,16 @@
...
   just --list
 
-# Run all checks from CI
-ci: format test
+# Run all pull request checks from CI via act
+ci:
+  act --container-architecture=linux/amd64 pull_request
+
+# Run Neovim nightly tests from CI via act
+ci-nightly:
```
Comment (johmsalas) [inline Justfile:10]: image

Comment (peterfication) [reply Justfile:10]: That's really strange. For me it looks like this:

Screenshot 2023-11-18 at 06 47 32

My act version is 0.2.54

Comment (peterfication) [reply Justfile:10]: Does it work for you when you specify the workflow file?

```
act --container-architecture=linux/amd64 -j TestsOnNightly -W .github/workflows/neovim-nightly.yml
```

Comment (johmsalas) [reply Justfile:10]: Was an issue in my local setup, git wasn't pulling the branch, therefore the file did not exist

**Review by johmsalas — APPROVED** [review]

**Pull request merged**